src/api/database/models/OngoingMeetings.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  AllowNull,
+  Column,
+  Table,
+  Model,
+  PrimaryKey
+} from "sequelize-typescript";
+import { STRING } from "sequelize";
+
+/*
+* This table maps TencentMeeting admin user ids with the number of ongoing meetings associated with the TM admin user ids. 
+*/
+
+@Table({
+  indexes: [{
+    unique: true,
+    fields: ['groupId', 'meetingId']
+  }, {
+    fields: ['groupId']
```

Comment:
groupId should be unique.

then you don't need the above unique index of groupId and meetingId.